cmd/launch-taskrun/main.go

```diff
@@ -235,6 +235,10 @@ type TaskRunConfig struct {
 	SingleComponent                 string `json:"SINGLE_COMPONENT"`
 	SingleComponentCustomResource   string `json:"SINGLE_COMPONENT_CUSTOM_RESOURCE"`
 	SingleComponentCustomResourceNs string `json:"SINGLE_COMPONENT_CUSTOM_RESOURCE_NS"`
+	VsaSigningKeySecretNs           string `json:"VSA_SIGNING_KEY_SECRET_NS"`
+	VsaSigningKeySecretName         string `json:"VSA_SIGNING_KEY_SECRET_NAME"`
+	VsaSigningKeySecretKey          string `json:"VSA_SIGNING_KEY_SECRET_KEY"`
```

Comment:
Isn't there a standard key that cosign is going to use for this?

```
  # generate a key-pair in Kubernetes Secret
  cosign generate-key-pair k8s://[NAMESPACE]/[NAME]
```

Could we use the same format as cosign and skip this additional config?

Reply:
Okay, I see how that we do have a default which is the same as what cosign uses. So disregard.